virt/virt-4-11-release-notes.adoc

```diff
@@ -181,10 +181,26 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-11-known-issues"]
 == Known issues
 
-//New known issues for 4.11 go above this comment
+* Some persistent volume claim (PVC) annotations created by the Containerized Data Importer (CDI) can cause the virtual machine snapshot restore operation to hang indefinitely. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070366[*BZ#2070366*])
+** As a workaround, you can remove the annotations manually:
+. Obtain the xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virtual-machine-snapshot-controller-and-custom-resource-definitions-crds[VirtualMachineSnapshotContent] custom resource (CR) name from the `status.virtualMachineSnapshotContentName` value in the `VirtualMachineSnapshot` CR.
+. Edit the `VirtualMachineSnapshotContent` CR by removing all lines that contain `k8s.io/cloneRequest`.
+. Optional: Delete the `DataVolume` and `PersistentVolumeClaim` objects in this namespace if their name begins with `restore-` and they are not referenced by virtual machines. This step is required if you did not specify a value for `spec.dataVolumeTemplates` in the `VirtualMachine` object.
+. Repeat the xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virt-restoring-vm-from-snapshot-cli_virt-managing-vm-snapshots[restore operation] with the updated `VirtualMachineSnapshot` CR.
+
+* The restore process of a cloned virtual machine (VM) snapshot fails if the source VM is deleted. This is because the `DataVolumeTemplate` value in the `VirtualMachineSnapshotContent` CR which refers to the persistent volume claim (PVC) of the source VM triggers a creation of clone data volume from a non existing source and is stuck waiting on the source PVC to be ready. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2104479[*BZ#2104479*])
```

Comment:
I'm having trouble following this second sentence, and I'm not sure that all of the technical details are necessary to get the point across. Maybe:

```suggestion
* The restore process of a cloned virtual machine (VM) snapshot fails if the source VM is deleted. This is because the `DataVolumeTemplate` value in the `VirtualMachineSnapshotContent` CR refers to the source VM's PVC, which no longer exists. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2104479[*BZ#2104479*])
```